backend/src/nodes/nodes/image/video_frame_iterator.py

```diff
@@ -251,3 +256,28 @@ def before(index: int):
             )
 
         await context.run_while(frame_count, before, fail_fast=True)
+
+        ffmpeg_reader.stdout.close()
+        ffmpeg_reader.wait()
+        if writer["out"] is not None:
+            writer["out"].stdin.close()
+            writer["out"].wait()
```

Comment:
Btw, I think these might help fix the video corruption stuff, potentially. It was necessary to add otherwise the video stream might not be closed before we try to mux the audio